From b4f19b01064d3a389e00302b054d990daf5c27fa Mon Sep 17 00:00:00 2001
From: Zhao Qiang <qiang.zhao@nxp.com>
Date: Tue, 19 Mar 2019 10:17:41 +0800
Subject: [PATCH 479/741] sdk: dts: remove dma-coherent from sdk dts for QE

commit aabe1efc16954940bd89e7ef8631429b5596f1be from
http://source.codeaurora.org/external/qoriq/qoriq-components/linux

dma-coherent feature in dts node will cause issue that
QE-HDLC received too little, when a lot of data is transmitted while
just little data received, the Tx buffer will run out.

Signed-off-by: Zhao Qiang <qiang.zhao@nxp.com>
Signed-off-by: Zhantao Tang <zhantao.tang@windriver.com>
---
 arch/arm64/boot/dts/freescale/fsl-ls1043a-rdb-sdk.dts | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-ls1043a-rdb-sdk.dts b/arch/arm64/boot/dts/freescale/fsl-ls1043a-rdb-sdk.dts
index 29c823266af9..5bfd2262d942 100644
--- a/arch/arm64/boot/dts/freescale/fsl-ls1043a-rdb-sdk.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-ls1043a-rdb-sdk.dts
@@ -197,10 +197,6 @@ pcie@3600000 {
 	dma-coherent;
 };
 
-&uqe {
-	dma-coherent;
-};
-
 &lpuart0 {
 	dma-coherent;
 };
-- 
2.17.1

